src/_pytest/config/__init__.py

```diff
@@ -47,6 +47,8 @@
 if TYPE_CHECKING:
     from typing import Type
 
+    _PluggyPlugin = object
+
```

Comment:
@bluetech since this might be used elsewhere - where should be put this?

Reply:
As long is we keep it internal for now (I mean, not intended to be exposed in the "public" typing in the future), I guess putting it where `PytestPluginManager` is defined makes most sense?

A short comment here would be helpful, something like

> A type to represent plugin objects. Plugins can be any namespace, so we can't narrow it down much, but we use an alias to make the intent clear. Ideally this type would be provided by pluggy itself.

BTW, I wouldn't put it under `TYPE_CHECKING`, this way we can use it without the annoying quotes.

Reply:
Thanks!  Amended in a39b4e2.